cql2/standard/clause_1_scope.adoc

```diff
@@ -6,3 +6,10 @@ This document defines
 
 * A text encoding for a CQL2 filter;
 * A JSON encoding for a CQL2 filter.
+
+NOTE: The CQL2 grammar contains many language elements of a general purpose, spatio-temporal expression language.
+The focus of this version of the language are boolean-valued filter expressions. It is planned to investigate, if CQL2
+should be extended through additional functions and operators to evolve into a backward-compatible expression language
+that is not restricted to boolean-valued expressions, but support expressions that result in other values, too, including
```

Comment:
Should read ... " but supports expressions that result in other values, too, ...".  Otherwise the addition is good.